docs/connectors.md

```diff
@@ -28,21 +28,28 @@ Source means the connector has the ability to get data from an upstream data sto
 | Connector | Source | Destination | Support |
 |-----------|-------|----|-------------|
 | [Algolia](https://github.com/ConduitIO/conduit-connector-algolia) | |✅ | Conduit |
+| [Airtable](https://github.com/conduitio-labs/conduit-connector-airtable) | | | Community |
```

Comment:
Nitpick: Airtable is alphabetically before Algolia 🙃

Reply:
Exactly. Thanks!